### PR TITLE
Use "github.com" hosted "drgn" repository

### PR DESCRIPTION
**Pull request by prakashsurya**

### CODE REVIEW

**Comment by prakashsurya** [comment]
- `linux-pkg-build` is [here](http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/linux-pkg-build/job/master/job/userland/job/pre-push/272/)

**Comment by prakashsurya** [comment]
cc: @sdimitro @pzakha

File: packages/drgn/config.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #
 
 # shellcheck disable=SC2034
-DEFAULT_PACKAGE_GIT_URL="https://gitlab.delphix.com/os-platform/drgn.git"
+DEFAULT_PACKAGE_GIT_URL="https://github.com/delphix/drgn.git"
 DEFAULT_PACKAGE_GIT_BRANCH="master"
```
Comment (pzakha) [inline packages/drgn/config.sh]: This is redundant, "master" is already the default.

Comment (prakashsurya) [reply packages/drgn/config.sh]: This is unrelated to my changes, so I'm dropping this. I recommend opening a bug and have it addressed in another PR.

**Comment by prakashsurya** [comment]
>  Would be worth it to do a git-ab-pre-push as well once the linux-pkg-build is done.

Yes, I can do this.

> Also, do you have plans to setup auto-update for this repository?

No concrete plans, but I believe we do want to have it be automatically updated. I believe our plans are to use the same model for this repository, as we do for "delphix/zfs"; i.e. we interact directly with upstream, and use auto-merges to keep our for up-to-date.

I don't know how this will be implemented, but I'd like to have that discussion elsewhere, since it's not directly related to this change.

**Comment by prakashsurya** [comment]
@pzakha can you give this another look; I think it's ready to go.

**Pull request merged**